src/main/java/com/arpnetworking/metrics/jvm/JvmMetricsRunnable.java

```diff
@@ -350,18 +389,22 @@ public Builder setCollectFileDescriptorMetrics(final Boolean value) {
         private MetricsFactory _metricsFactory;
         private ManagementFactory _managementFactory = MANAGEMENT_FACTORY_DEFAULT;
         private Boolean _swallowException = true;
-        private Boolean _collectNonHeapMemoryMetrics = true;
+        private Boolean _collectNonHeapMemoryMetrics = false;
```

Comment:
This is technically not backwards compatible. Just to confirm the metrics produced by the pool collector and the non-heap collector are identical (e.g. name + value)? -- if so this is fine.

Reply:
The only difference is that some metrics erroneously had non_heap in the name. They have been fixed. If a user had explicitly set the non heap collector, that collector will only get the non heap metrics, but they will be defaulted into the now fixed heap metrics as well from the pool collector.

Reply:
If a user enables both, the non-heap collector and the pool usage collector, won't the non-heap ones be emitted at higher frequency? Doesn't seem like a big issue, since there's a deprecation warning on there. Looks good otherwise.

Reply:
Yes that's true.  I'm considering releasing this as  Y change to indicate that there is a little more than just a bug fix here.